MAUI/Scheduler/resource-view.md

```diff
@@ -0,0 +1,375 @@
+</schedule:SfScheduler.ResourceView>
+</schedule:SfScheduler>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+this.Scheduler.ResourceView.MinimumRowHeight = 100;
+{% endhighlight %}
+{% endtabs %} 
+
+N>
+* By default, if the viewport height is greater than 400 then each resource height will be calculated by viewport size divided by the minimum value of scheduler resources count and 4 (default resource count). 
+* If the viewport height is lesser than 400 then each resource height will be calculated by default viewport size(4 (default resource*100)) divided by the minimum value of scheduler resources count and 4 (default resource count).
+* If the MinimumRowHeight is less than the default row height then the default row height will be used.
+
+## Assign resources to special time regions
+You can highlight a resource’s availability by creating special time regions in the timeline day, timeline week, and timeline workweek views.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainWindow.xaml" %}
+
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="9" %}
+this.Scheduler.DaysView.TimeRegions.Add(new SchedulerTimeRegion
+{
+	StartTime = new DateTime(2022, 06, 04, 13, 0, 0),
+	EndTime = new DateTime(2022, 06, 04, 14, 0, 0),
+	Text = "Lunch",
+	RecurrenceRule = "FREQ=DAILY;INTERVAL=1",
+	ResourceIds = new ObservableCollection<object>() { "1000", "1001", "1002" }
+});
+{% endhighlight %}
+{% endtabs %}
+
+## Programmatic resource selection
+You can programmatically select the resource by using the `SelectedResourceId` and `SelectedDate` of the SfScheduler. Please [click]() here to see more details about programmatic date selection.
+
+## Appearance customization
+
+The resource appearance customization can be achieved by using the [ResourceHeaderTemplate]() and [TextStyle]() properties of the [SchedulerResourceView]().
+
+#### Customize resource appearance using text style
+The resource header text style can be customized by using the [TextStyle]() property of the [SchedulerResourceView]().
+	
+#### Customize resource appearance using ResourceHeaderTemplate
+The resource appearance customization can be achieved by using the [ResourceHeaderTemplate]() property of the [SchedulerResourceView]().
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="28" %}
+
+<ContentPage.Behaviors>
+    <local:ResourceViewBehavior/>
+</ContentPage.Behaviors>
+    
+<Grid>
+    <schedule:SfScheduler x:Name="Scheduler"  View="TimelineMonth"
+                                AppointmentsSource="{Binding Events}"
+                                AllowedViews="TimelineDay,TimelineMonth,TimelineWeek,TimelineWorkWeek" >
+
+        <schedule:SfScheduler.Resources>
+            <local:SfImageSourceConverter x:Key="imageConverter"/>
+        </schedule:SfScheduler.Resources>
+        <schedule:SfScheduler.ResourceView>
+            <schedule:SchedulerResourceView Resources="{Binding Resources}">
```

Comment:
Add c# code for binding class

Resources="{Binding Resources}" , AppointmentsSource="{Binding Events}"